packages/grafana-data/src/field/displayProcessor.test.ts

```diff
@@ -16,9 +16,10 @@ function getDisplayProcessorFromConfig(config: FieldConfig) {
 function assertSame(input: any, processors: DisplayProcessor[], match: DisplayValue) {
   processors.forEach(processor => {
     const value = processor(input);
-    expect(value.text).toEqual(match.text);
-    if (match.hasOwnProperty('numeric')) {
-      expect(value.numeric).toEqual(match.numeric);
+    for (const key in match) {
+      if (match.hasOwnProperty(key)) {
```

Comment:
NIT: I think this could be simplified with `for ..of` as then `hasOwnProperty` check won't be needed - `for (const key of Object.keys(match))`.

Reply:
yea, maybe. Just never liked creating a new array just to be able to loop over properties

Reply:
but agree that is probably the preferred/best practice method